explore-analyze/discover/discover-get-started.md

```diff
@@ -303,6 +305,35 @@ Note that in ES|QL mode, the **Documents** tab is named **Results**.
 
 Learn more about how to use ES|QL queries in [Using ES|QL](try-esql.md).
 
+### Run multiple explorations with tabs
+```{applies_to}
+stack: preview 9.2
+serverless: preview
+```
+**Discover** supports multiple tabs to help you explore different aspects of your data simultaneously. Each tab maintains its own independent state, including the query ({{esql}} or KQL), time range, filters, selected data source, columns and sort order you defined, and the active [context-aware experience](#context-aware-discover).
+
+This allows you to pivot quickly between different explorations without rebuilding everything. For example:
+
+* **Compare time periods:** Open multiple tabs with the same query but different time ranges
+* **Test query variations:** Duplicate a tab to experiment with different {{esql}} queries or filters
+* **Switch contexts:** Keep separate tabs for logs, metrics, and traces exploration
+* **Test a hypothesis:** Switch between different data sources or field combinations
+
+#### Manage Discover tabs
+
+You can open new tabs or duplicate existing ones to compare different queries:
+- To start a fresh exploration in a new tab, select the {icon}`plus` icon next to existing tabs.
+- To test variations of your current query in a new tab, hover over a tab and select the {icon}`boxes_vertical` **Actions** icon, then select **Duplicate**.
+
+To manage and organize your tabs, you can: 
+- Rename them: Hover over a tab and select the {icon}`boxes_vertical` **Actions** icon, then select **Rename**.
+- Reorder them: Drag and drop a tab to move it.
+- Close them: Hover over a tab and select the {icon}`cross` icon.
+- Close several tabs at once: When you hover over a tab and select the {icon}`boxes_vertical` **Actions** icon, options let you **Close other tabs** to keep only the active tab open or **Close tabs to the right** to only keep your first tabs and discard any subsequent tabs.
+- Reopen recently closed tabs: If you close a tab by mistake, you can retrieve it by selecting the {icon}`boxes_vertical` **Tabs bar menu** icon located at the end of the tab bar.
+
+To keep all of your tabs ready for a later exploration session, you can [Save your Discover session](#save-discover-search). All currently open tabs are saved within the session and will be there when you open it later.
```

Comment:
```suggestion
To keep all of your tabs for later, you can [Save your Discover session](#save-discover-search). All currently open tabs are saved within the session and will be there when you open it again.
```
Nit: I felt "exploration session" followed by "Discover session" flowed oddly, then I changed it and thought "later" twice had a weird flow too 😅 Minor nit though, fine to leave as is too.